src/javascript/app/pages/trade/purchase.js

```diff
@@ -134,6 +134,19 @@ const Purchase = (() => {
                                 additional_message = localize('Try our other markets.');
                             }
                             message = `${error.message}. ${additional_message}`;
+                        } else if (/ClientUnwelcome/.test(error.code) && /gb/.test(State.getResponse('authorize.country'))) {
+                            let message_text = '';
+                            let additional_message = '';
+
+                            if (!Client.hasAccountType('real') && /^VRTC/.test(State.getResponse('authorize.loginid'))) {
+                                message_text = localize('Please complete the [_1]Real Account form[_2] to verify your age as required by the <strong>UK Gambling</strong> Commission (UKGC).', [`<a href='${urlFor('new_account/realws')}'>`, '</a>']);
+                                additional_message = localize('Once verified, you will be able to trade on both real and virtual accounts.');
+                            } else if (Client.hasAccountType('real') && /^MX|VRTC/.test(State.getResponse('authorize.loginid'))) {
```

Comment:
we don't want to check the broker code unless we have to, it's always better to check landing company or existing functions. and for responses of authorize, they are all stored in Client so we should get them from there. in this case my suggestion is to use something similar to `/^virtual|iom$/i.test(Client.get('landing_company_shortcode')` instead.